### DISCUSSION TITLE
What are the supposed files extensions for write_voxel_grid?

### DISCUSSION THREAD

**Discussion by tempaccountforissue** [isl-org/Open3D#5928] [General]
In the docs, it lists the supported file extensions for pointclouds, meshes, and images. However, it does not list the supported file extensions for voxel grids. What are the supported file extensions for voxel grids?

**Comment by Wissotsky** [comment]
According to this code it only supports **.ply** files(both reading and writing)

https://github.com/isl-org/Open3D/blob/1e7b17438687a0b0c1e5a7187321ac7044afe275/cpp/open3d/io/VoxelGridIO.cpp#L20-L35
